web-ui/src/main/resources/catalog/components/common/map/mapsManager.js

```diff
@@ -197,32 +203,37 @@
               }
             }
 
-            // layers
+            // load layers from Settings
```

Comment:
Plus, it didn't consider layers from settings as background layers but it didn't show them on the layer list. It was weird, that's also fixed.